discord/index.py

```diff
@@ -77,6 +78,24 @@ async def top_stop_error(ctx, error):
     if isinstance(error, MissingPermissions):
         return await ctx.reply('`Missing Permissions`') 
     else:
-        pass           
+        pass 
+
+async def _check_called(ctx, state):
+    if state: 
+        msg = await ctx.reply('`Session Already Active`')
+        await msg.delete(delay=5)
+        return state
+help_dict = {
+            "**Top Scites**": "<@&846783290332413964>`top`", 
```

Comment:
Maybe we could rename this to `start` instead of `top`; then it links with `stop` nicely :)

Reply:
Done!